source/opt/trim_capabilities_pass.cpp

```diff
@@ -368,6 +368,13 @@ void TrimCapabilitiesPass::addInstructionRequirements(
     return;
   }
 
+  // Ignoring OpBeginInvocationInterlockEXT and OpEndInvocationInterlockEXT
```

Comment:
nit: maybe this should be inside the `addInstructionRequirementsForOpcode`?

Reply:
Done.